rust/arrow/src/compute/kernels/cast.rs

```diff
@@ -1297,6 +1301,57 @@ fn cast_list_inner<OffsetSize: OffsetSizeTrait>(
     Ok(Arc::new(list) as ArrayRef)
 }
 
+/// Helper function to cast from Utf8 > LargeUtf8 and vice versa. If the LargeUtf8 it too large for
+/// a Utf8 array it will return an Error.
```

Comment:
```suggestion
/// Helper function to cast from `Utf8` to `LargeUtf8` and vice versa. If the `LargeUtf8` is too large for
/// a `Utf8` array it will return an Error.
```